vllm/v1/worker/xpu_model_runner.py

```diff
@@ -27,7 +27,7 @@ def __init__(
         self.cascade_attn_enabled = False
 
     def _init_device_properties(self) -> None:
-        pass
+        self.num_sms = None
```

Comment:
On top of this change, may I suggest an improvement to move these customization to `gpu_model_runner.py` as these are just backend specific dispatch logic which can be handled easier without class inheritance? See proposal here:
* https://github.com/vllm-project/vllm/pull/20698

Reply:
Actually these function was in gpu_model_runner.py originally and move to different device model_runner for cleaner readness. So I think we could follow this design.